spec/requests/portfolio_items_spec.rb

```diff
@@ -86,21 +86,64 @@
   end
 
   describe 'DELETE admin tagged /portfolio_items/:portfolio_item_id' do
-    # TODO: https://github.com/ManageIQ/catalog-api/issues/85
-    let(:valid_attributes) { { :name => 'PatchPortfolio', :description => 'description for patched portfolio' } }
-
     context 'when v1.0 :portfolio_item_id is valid' do
       before do
-        delete "#{api}/portfolio_items/#{portfolio_item_id}", :headers => default_headers, :params => valid_attributes
+        delete "#{api}/portfolio_items/#{portfolio_item_id}", :headers => default_headers
       end
 
       it 'discards the record' do
-        expect(response).to have_http_status(204)
+        expect(response).to have_http_status(:ok)
       end
 
       it 'is still present in the db, just with deleted_at set' do
         expect(PortfolioItem.with_discarded.find_by(:id => portfolio_item_id).discarded_at).to_not be_nil
       end
+
+      it 'returns the restore_key in the body' do
+        expect(json["restore_key"]).to_not be_nil
```

Comment:
This expectation seems kinda unnecessary since we've already got the expectation on line 104, unless you think there's a good reason to believe that the `Digest::SHA1.hexdigest` call is going to generate nil and be equal to the restore key?